finalPrep/README.md

```diff
@@ -18,37 +18,37 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- [1](https://user-images.githubusercontent.com/90354031/134787262-4c36402d-f1eb-4236-9c1f-d218862fec8c.png)
+- [2](https://user-images.githubusercontent.com/90354031/134787289-bef1dcef-01f5-49ce-9aae-a927b969ff8a.png)
+- [3](https://user-images.githubusercontent.com/90354031/134787331-570f1269-4352-4666-9835-3652e53e9f1d.png)
```

Comment:
Good start to your schedule. I suggest adding an intentional time to evaluate how you did your previous week, and adjusting for the upcoming week. You can also sync your calendar with one on your [program's homepage](https://frontend.turing.edu/).

Reply:
Done! (What's the order of operations typically on something like this? Do I resolve this convo?)